Sources/Paywalls/Components/PaywallButtonComponent.swift

```diff
@@ -21,13 +21,38 @@ public extension PaywallComponent {
 
     struct ButtonComponent: PaywallComponentBase {
 
+        // swiftlint:disable:next nesting
+        public enum Action: Codable, Sendable, Hashable, Equatable {
+            case restorePurchases
+            case navigateTo(destination: Destination)
+            case navigateBack
+        }
+
+        // swiftlint:disable:next nesting
+        public enum Destination: Codable, Sendable, Hashable, Equatable {
+            case customerCenter
+            case URL(urlLid: String, method: URLMethod)
+            case privacyPolicy(urlLid: String, method: URLMethod)
+            case terms(urlLid: String, method: URLMethod)
```

Comment:
The `privacyPolicy` and `terms` cases are the same as the `URL` case, but I included them since our model mirrors the dashboard. (Same reason why we have the `deepLink` case in the `URLMethod` enum, even though it is the same as `externalBrowser`.) Let me know if you have any thoughts on this.

Reply:
This looks good to me! 👍